src/main/java/io/galeb/manager/entity/Provider.java

```diff
@@ -74,4 +74,8 @@ public Provider setProvisioning(String provisioning) {
         return this;
     }
 
+    @Override
+    public String getEnvName() {
+        return null;
```

Comment:
There's a NullPointerException risk. Using generic constant may be a best approach, not?